lib/openstack_api/openstack_query.py

```diff
@@ -19,7 +19,6 @@
 
 
 class OpenstackQuery(OpenstackWrapperBase):
```

Comment:
Could we add a docstring to describe what this class does?

Reply:
this class is the deprecated class - when the updated openstack_query stuff is approved I will remove this